TKGame/BackEnd/EntityManager.cs

```diff
@@ -147,5 +150,16 @@ public static void DamageEnemy()
                 }
             }
         }
+
+        public static void EnemyDamage()
+        {
+            foreach(Entity entity in entities)
+            {
+                if (entity != entities[0] && entity.hitBox.Intersects(entities[0].hitBox) && entity.isEnemy)
+                {
+                    entities[0].health -= 10;
```

Comment:
This is gonna happen once every frame so at least in my case 240 times in a second so won't this instantly kill the player?
We should probably limit the number of times the damage can be inflicted. See the TimeElapsed and related usage for the fire powerups for reference.

Reply:
I fixed this with a simple counter int in then function and only decrementing  the health once a second.